internal/gatewayapi/resource.go

```diff
@@ -20,41 +20,6 @@ import (
 
 type XdsIRMap map[string]*ir.Xds
 type InfraIRMap map[string]*ir.Infra
-type GatewayClassResources map[string]*Resources
```

Comment:
list will hit a similar issue
```bash
panic: watchable.DeepCopy: type is not copiable: *[]*gatewayapi.Resources
```

Reply:
yeah I had to write my own `DeepCopy` as well